### ISSUE TITLE
How to get the distribution of the gradient norm?

### ISSUE THREAD

**Issue by dpengwen** [libuyu/GHM_Detection#21]
Hi,
  Thanks for your great work! I have a problem that how to get the distribution of the gradient norm as shown in Fig.2.  In the paper, it says the distribution from is from a converged one-stage detection model.  Is the converged model trained with the gradient harmonizing mechanism?  What dataset is used for the statistics, the training set of COCO or the testing set of COCO?

Sorry for bothering you!
